.github/workflows/push-on-branch.yml

```diff
@@ -0,0 +1,61 @@
+name: Lint, Test, Build and Push on Branches
+
+on:
+  push:
+    branches: [ main, develop ]
+
+env:
+  ECR_REPOSITORY: naturescot/fit-and-competent
```

Comment:
Copy-pasta err!

Reply:
Resolved in https://github.com/Scottish-Natural-Heritage/trap-registration-return/pull/1/commits/99ad45ca3f58f2088352e0132c17a64a12ffccf5.